server/routes/trial-management/add-panel-members/add-panel-member.controller.js

```diff
@@ -0,0 +1,199 @@
+(function() {
+  'use strict';
+
+  const  _ = require('lodash')
+    , addPanelMembersValidator = require('../../../config/validation/add-panel-members')
+    , { addPanelMembersDAO, availableJurorsDAO } = require('../../../objects/panel')
+    , poolsValidator = require('../../../config/validation/generate-panel-pools')
+    , validate = require('validate.js');
```

Comment:
could we split these each into its own const declaration?

Reply:
changed